website/src/controllers/api-v3/groups.js

```diff
@@ -563,15 +520,25 @@ api.inviteToGroup = {
     let uuids = req.body.uuids;
     let emails = req.body.emails;
 
-    if (uuids && emails) { // TODO fix this, low priority, allow for inviting by both at the same time
-      throw new BadRequest(res.t('canOnlyInviteEmailUuid'));
-    } else if (Array.isArray(uuids)) {
-      // return _inviteByUUIDs(uuids, group, user, req, res, next);
-    } else if (Array.isArray(emails)) {
-      // return _inviteByEmails(emails, group, user, req, res, next);
-    } else {
+    if (!Array.isArray(uuids) && !Array.isArray(emails)) {
```

Comment:
And, to clarify, here is the logic I am about to refactor the code for:

//If we have neither, return error
//If we have uuids, but empty or object, return error
 //If we have emails, but empty or object, return error